apps/concierge_site/test/web/controllers/auth_controller_test.exs

```diff
@@ -67,6 +71,7 @@ defmodule ConciergeSite.Web.AuthControllerTest do
          %{conn: conn, rider: %User{id: id}} do
       new_email = "newemail@example.com"
       new_phone_number = "+15559876543"
+      new_role = "admin"
```

Comment:
**suggestion:** it might be nice to have an explicit case for the "user" role case, since the failure where that isn't handled correctly and results in improperly assigning an "admin" role would be bad.

Reply:
Yep, that's a very good idea. Added a test for that case.